pandas/core/algorithms.py

```diff
@@ -488,12 +492,12 @@ def f(c, v):
             f = np.in1d
 
     else:
-        common = np.find_common_type([values.dtype, comps.dtype], [])
+        common = np.find_common_type([values.dtype, comps_array.dtype], [])
         values = values.astype(common, copy=False)
-        comps = comps.astype(common, copy=False)
+        comps_array = comps_array.astype(common, copy=False)
         f = htable.ismember
 
-    return f(comps, values)
+    return f(comps_array, values)
```

Comment:
I'm surprised that your changes didn't remove any mypy ignores inside this function.

Reply:
I think that is because  `extract_array` returned `Any` before, so no ignores were necessary.